examples/dt/bgp/edpm/nodeset/values.yaml

```diff
@@ -89,17 +89,12 @@ data:
             - ip_netmask: {{ lookup('vars', 'bgpmainnetv6_ip') }}/128
         edpm_nodes_validation_validate_controllers_icmp: false
         edpm_nodes_validation_validate_gateway_icmp: false
-        edpm_selinux_mode: enforcing
```

Comment:
ohkk variable is renamed to edpm_bootstrap_selinux_mode and defaults to enforcing